### ISSUE TITLE
Self-Signed SSL, REST Open API url

### ISSUE THREAD

**Issue by xevilmaxx** [riok/Kreya#168]
**Describe the bug**
I try to reach my intranetwork website which exposes Open API json, it uses only SSL but has a self-signed certificate.
Like this:
https://10.169.168.52:444/swagger/v1/swagger.json

ERROR:
```
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure: RemoteCertificateNameMismatch, RemoteCertificateChainErrors
   at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)
   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](Boolean receiveFirst, Byte[] reAuthenticationData, CancellationToken cancellationToken)
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(QueueItem queueItem)
   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.HttpConnectionWaiter`1.WaitForConnectionAsync(Boolean async, CancellationToken requestCancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at Microsoft.Extensions.Http.Logging.LoggingHttpMessageHandler.<SendAsync>g__Core|5_0(HttpRequestMessage request, CancellationToken cancellationToken)
   at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<SendAsync>g__Core|5_0(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at Kreya.Rest.Core.Importer.OpenApiUrlImporter.ImportOpenApiDocument(OpenApiUrlImporterOptions options) in C:\dev\gitlab-runner\builds\ifF5m2AW\0\riokch\kreya\kreya\src\rest\Kreya.Rest.Core\Importer\OpenApiUrlImporter.cs:line 78
```

**To Reproduce**
1. Go to 'Importers'
2. Click on 'New Importer of Open API URL'
3. Type url of openapi.json
4. Save
5. Error

**Expected behavior**
Imported Open API (which works if i copy paste it to a file and import it from file)

**Screenshots**

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS: Windows 11
 - Kreya Version 1.11.1

**Additional context**
You probably need to add this solution to your code:
[https://stackoverflow.com/questions/52939211/the-ssl-connection-could-not-be-established](https://stackoverflow.com/questions/52939211/the-ssl-connection-could-not-be-established)

**Comment by CommonGuy** [comment]
Thanks for the bug report, this is not yet supported. We will add an option to disable TLS certificate validation in the OpenAPI importer.

**Comment by CommonGuy** [comment]
This should be fixed with the newly released 1.12 version.
